plan4past/utils/to_pddl.py

```diff
@@ -0,0 +1,138 @@
+"""Converts a PDDL object to a PDDL string."""
```

Comment:
Do we need this module?

Reply:
It appears to be unused. It will be removed, thanks